src/java/org/apache/cassandra/db/marshal/PartitionerDefinedOrder.java

```diff
@@ -97,8 +98,17 @@ public String toJSONString(ByteBuffer buffer, ProtocolVersion protocolVersion)
     }
 
     @Override
-    public ByteSource asComparableBytes(ByteBuffer buf, ByteComparable.Version version)
+    public ByteSource asComparableBytes(ByteBuffer buf, Version version)
     {
+        if (version != Version.LEGACY)
+        {
+            // For ByteComparable.Version.DSE68 and above we encode an empty key with a null byte source. This
```

Comment:
There are obsolete version names in this comment.

Reply:
ah, thanks, let me change that